src/line.rs

```diff
@@ -17,7 +16,7 @@ pub struct Line {
     pub text: Option<String>,
     #[serde(rename="cursor")]
     pub cursors: Option<Vec<u64>>,
-    pub styles: Option<Vec<i64>>,
+    pub styles: Vec<i64>,
```

Comment:
In practice it seems to be the case but I'll ask on irc.
It makes the code more readable if we can get rid of `Option`, so that we don't have to write any access to styles with `if let Some(styles) = self.styles`/`self.styles.ok_or`/etc.

Same thing for the cursors btw.